src/Workspaces/VisualBasic/Portable/Classification/Worker.vb

```diff
@@ -85,6 +85,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Classification
 
                 If type IsNot Nothing Then
                     AddClassification(token.Span, type)
+
+                    ' Additionally classify static symbols
+                    If (token.Kind() = SyntaxKind.IdentifierToken _
+                        AndAlso ClassificationHelpers.IsStaticallyDeclared(token)) Then
```

Comment:
unnecessary parens.  Forcing the wrapping in this style in VB is especially egregious (in the IDE codebase at least) because then you're forced into needing the line wrapping char.